src/locales/en.json

```
@@ -89,6 +89,7 @@
   "Scan ASN to start receiving": "Scan ASN to start receiving",
   "Scan barcodes to receive them": "Scan barcodes to receive them",
   "Scan items": "Scan items",
+  "Scanned item is not present within the shipment": "Scanned item is not present within the shipment: {itemName}",
```

```suggestion
  "Scanned item is not present within the shipment:": "Scanned item is not present within the shipment: {itemName}",
```

Also we can remove "Failed to scan" from this file as it will not be used anymore.